api/control/v1alpha1/types.go

```diff
@@ -443,6 +457,42 @@ const (
 // Type checking errors are posted to the status.Errors of the Binding and BindingPolicy.
 type Expression string
 
+// Augmentation defines what is implicitly queryable in the CEL expressions.
+// This means that while the listed fields are not part of the object's schema,
+// they can be referred to in the CEL expressions.
+//
+// Besides for Augmentations, the other objects queryable in CEL expressions are:
+//
+//  1. The workload object which resides in the WDS, as `obj`;
+//
+//  2. The following subresources of the propagated object at a WEC, as `returned`:
+//
+//     - .status: the status of the object at the WEC.
+type Augmentation struct {
+	// `inventory` is the inventory object reference.
+	// It contains information about the inventory object (WEC).
+	Inventory InventoryObjectReference `json:"inventory"`
+	// `propagation` is the last reported metadata about one object at one WEC.
+	// It contains information about the propagation of the object.
+	Propagation PropagationMeta `json:"propagation"`
+}
+
+// InventoryObjectReference identifies an inventory object.
+// Inventory is queryable in StatusCollector expressions using the `inventory` object.
+type InventoryObjectReference struct {
+	// `name` is the inventory object's name
+	Name string `json:"name"`
+}
+
+// PropagationMeta is the last reported metadata about one workload object at one WEC.
+// PropagationMeta is queryable in StatusCollector expressions using the `propagation` object.
+type PropagationMeta struct {
+	// `infoAgeSeconds` is the age of the object in seconds.
```

Comment:
With respect to a given WEC, an object has two ages: its age in the WDS and its age in the WEC.

My original intent for _info_ age is to characterize how old is the _information_ (about the object) that is in the core but originated in the WEC. How old is the core's copy of the object status and klusterlet success/fail/whatever from the WEC?

Reply:
This is moot now.